viam-cartographer.go

```diff
@@ -241,8 +245,10 @@ func New(
 		cartoFacadeTimeout:            cartoFacadeTimeout,
 		localizationMode:              mapRateSec == 0,
 		mapTimestamp:                  time.Now().UTC(),
-		jobDone:                       false,
 	}
+
+	cartoSvc.jobDone.Store(false)
```

Comment:
I think you can remove this given that the zero value of an atomic bool is already false, so this should already be set to false when the `CartographerService{}` struct is created above.